src/typescript/05-exporting-and-importing/exporting-and-importing.md

```diff
@@ -13,6 +13,7 @@ In this section, you will:
 - Export values from a file
 - Import values from another file
 - Learn why named exports are preferred over default exports
+- For the exercise, we will learn how to import and export functions between modules.
```

Comment:
All of the bullet points should finish the sentence “In this section, you will:”